examples/README.md

```diff
@@ -0,0 +1,2 @@
+# Examples
+Basic simulation of the SEIR Model: https://nbviewer.jupyter.org/github/SABS-R3-Epidemiology/seirmo/blob/issue-20-jupyter-notebook-SEIR-model/seirmo/examples/basic_simulation.ipynb
```

Comment:
Could you update the new link please?

https://nbviewer.jupyter.org/github/SABS-R3-Epidemiology/seirmo/blob/issue-20-jupyter-notebook-SEIR-model/examples/SEIR_simulation.ipynb